src/web/src/components/DynamicImage.tsx

```diff
@@ -31,8 +31,8 @@ const useStyles = makeStyles((theme) => ({
     right: 0,
     boxSizing: 'border-box',
     margin: 0,
-    backgroundColor: theme.palette.type === 'light' ? '#E5E5E5' : '#000000',
-    opacity: '.35',
+    backgroundColor: '#000000',
```

Comment:
Let's add this colour to the theme

Reply:
I don't want to connect the landing page with the theme colors now, let's make this when implementing a dark/light themes in the landing page. For now, let's land the new land page without a dark/light theme.